modules/fb_apache/lib/logstash_registry.rb

```diff
@@ -0,0 +1 @@
+LogStash::PLUGIN_REGISTRY.add(:modules, "fb_apache", LogStash::Modules::Scaffold.new("fb_apache", File.join(File.dirname(__FILE__), "..", "configuration")))
```

Comment:
We don't need that file because we actually load the module from a logstash directory and not from a gem.